actions/main/src/main.sh

```diff
@@ -20,6 +20,10 @@ parseInputs () {
   if [ -n "${INPUT_SECRET_ID}" ]; then
     export secret_id=${INPUT_SECRET_ID}
   fi
+  SONAR_TOKEN=""
+  if [ -n "${INPUT_SONAR_TOKEN}" ]; then
+    export SONAR_TOKEN=${INPUT_SONAR_TOKEN}
+  fi
```

Comment:
What is the point of the `-n` check here? If `INPUT_SONAR_TOKEN` is empty then wouldn't the result of
```
export SONAR_TOKEN=${INPUT_SONAR_TOKEN
```
be the same as
```
export SONAR_TOKEN=""
```

So I would instead write
```suggestion
  export SONAR_TOKEN="${INPUT_SONAR_TOKEN}"
```

Note that if you had a default that you wanted to use if a variable wasn't defined, you can write it this way:
```
export SONAR_TOKEN="${INPUT_SONAR_TOKEN:default-token}"
```
(where `default_token` is the actual token and not another shell variable)

Reply:
That's a good point - I don't need a default value, so I'll use your first suggestion!